presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -342,6 +359,16 @@ public QueryInfo createQuery(SessionSupplier sessionSupplier, String query)
         QueryExecution queryExecution;
         Statement statement;
         try {
+            if (!acceptQueries.get()) {
+                int activeWorkerCount = internalNodeManager.getNodes(ACTIVE).size();
```

Comment:
I guess we should also handle the case for `node-scheduler.include-coordinator=true` similar to how we handle it [here](https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/server/ClusterStatsResource.java#L57-L60).